File: tests/test_code/rb/simple_b/simple_b.rb
"""
Comments
"""


def a()
    b()
end

# comments
def b()
    a("""STRC #""")
end

class Cls
    def initialize(val)
        @val = val
    end
    def d(a="String")
        a("AnotherSTR")
    end
end

c = Cls.new()
c.d()

File: tests/test_code/php/simple_b/simple_b.php
<?php

require __DIR__.'/../vendor/autoload.php';

function a($param) {
    b($param);
}


function b() {
    a("STRC #");
}


class C {
    function d($param) {
        a("AnotherSTR");
    }
}

$c = new C();
$c->d();

?>
